frontend/src/app/register/page.tsx

```diff
@@ -0,0 +1,88 @@
+"use client";
+
+import { useEffect } from "react";
+import styles from "./register.module.css";
+import { useSearchParams } from "next/navigation";
+import axios from "@/axios/client";
+import { ResponseCredential, ServerData } from "@/types/webAuthn";
+
+export default function RegisterUser() {
+  const searchParams = useSearchParams();
+
+  useEffect(() => {
+    async function f() {
+      // Decode token & check validity
+      const token = searchParams.get("token");
+
+      // TODO: verify token is valid in backend during registerRequest
```

Comment:
Will be able to implement once we have emails / available_tokens table in backend. I think the verification logic should go in registerRequest.